Dockerfile.slim

```diff
@@ -0,0 +1,57 @@
+FROM clojure:lein-alpine
+RUN apk add --no-cache make
+# Install only dependencies
+WORKDIR /app
+COPY project.clj .
+COPY resources/puppetlabs/puppetdb/bootstrap.cfg \
+    ./resources/puppetlabs/puppetdb/bootstrap.cfg
+RUN lein with-profile uberjar deps
+# Build uberjar -- see .dockerignore
+COPY . .
+RUN lein with-profile uberjar uberjar
+
+
+# FROM openjdk:8-jre-alpine
+FROM alpine-jre-puppet
```

Comment:
This is the `puppet/puppet-agent-alpine` image but built from `openjdk:8-jre-alpine` instead of `alpine:3.8` (https://github.com/puppetlabs/puppet-agent/blob/master/docker/puppet-agent-alpine/Dockerfile#L114).

This only exists on my machine at the moment.